unstract/connectors/src/unstract/connectors/filesystems/azure_cloud_storage/azure_cloud_storage.py

```diff
@@ -65,9 +64,9 @@ def get_fsspec_fs(self) -> AzureBlobFileSystem:
     def test_credentials(self) -> bool:
         """To test credentials for Azure Cloud Storage."""
         try:
-            is_dir = bool(self.get_fsspec_fs().isdir(self.bucket))
+            is_dir = bool(self.get_fsspec_fs().isdir(""))
```

Comment:
@chandrasekharan-zipstack what if the permissions provided are restrictive in nature and the access is only for particular bucket. Will this test connection still work? Should we may be include the path also in to this checks?

Reply:
@ritwik-g we could take up testing access based on paths once we unify the BE / FE for the 2 columns (connector and source / dest configuration)